Buddies.Web/src/pages/leaderboard.tsx

```diff
@@ -0,0 +1,175 @@
+import * as React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import Avatar from '@mui/material/Avatar';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import { Box, Button, Container } from '@mui/material';
+import EmailIcon from '@mui/icons-material/Email';
+import TablePagination from '@mui/material/TablePagination';
+import { useEffect } from 'react';
+import CardActionArea from '@mui/material/CardActionArea';
+import { getLeaderboard } from '../api';
+
+type User = {
+  firstName: string,
+  lastName: string,
+  email: string,
+  userId: number,
+  buddyScore: number
+};
```

Comment:
We have a `UserInfoResponse` in `src/api/model/userInfoResponse.ts`. You can run `yarn gen-types` to regenerate the type to include `buddyScore`.